syntax = "proto3";

package bazel_scratch;

// DO NOT USE.
message FrameMetadataTestOnly {
    int64 timestamp_ns = 1;

    int64 exposure_time_ns = 2;

    // Linear gain.
    float gain = 3;
}
